packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -69,42 +69,52 @@ class Checkbox extends StatelessWidget {
   static const double width = 18.0;
 
   @override
+  _CheckboxState createState() => new _CheckboxState();
+}
+
+class _CheckboxState extends State<Checkbox> with TickerProviderStateMixin {
+  @override
   Widget build(BuildContext context) {
     assert(debugCheckHasMaterial(context));
     ThemeData themeData = Theme.of(context);
     return new _CheckboxRenderObjectWidget(
-      value: value,
-      activeColor: activeColor ?? themeData.accentColor,
-      inactiveColor: onChanged != null ? themeData.unselectedWidgetColor : themeData.disabledColor,
-      onChanged: onChanged
+      value: config.value,
+      activeColor: config.activeColor ?? themeData.accentColor,
+      inactiveColor: config.onChanged != null ? themeData.unselectedWidgetColor : themeData.disabledColor,
+      onChanged: config.onChanged,
+      tickerProvider: this,
```

Comment:
To make that work (without requiring every widget to also have a custom element) we'd have to bake knowledge about Tickers into framework.dart, which I would be very reluctant to do.